book/src/installation-source.md

```diff
@@ -26,35 +27,45 @@ directory will be the location you cloned Lighthouse to during the installation
 - `git checkout ${VERSION}`
 - `make`
 
+
 ## Detailed Instructions
 
-1. Install Rust and Cargo with [rustup](https://rustup.rs/).
-    - Use the `stable` toolchain (it's the default).
-	- Check the [Troubleshooting](#troubleshooting) section for additional
-		dependencies (e.g., `cmake`).
+1. Install the build dependencies for your platform
+	- Check the [Dependencies](#dependencies) section for additional
+		information.
 1. Clone the Lighthouse repository.
     - Run `$ git clone https://github.com/sigp/lighthouse.git`
     - Change into the newly created directory with `$ cd lighthouse`
 1. Build Lighthouse with `$ make`.
-1. Installation was successful if `$ lighthouse --help` displays the
-   command-line documentation.
+1. The build was successful if the `lighthouse` binary exists in the `target/release` directory and `$ ./target/release/lighthouse --help` displays the command-line documentation.
```

Comment:
On my Windows box it also installs it to `C:\Users\Me\.cargo\bin\lighthouse.exe`, which is added to the `$PATH` by `rustup`, i.e. I think we can simplify this and leave it as `lighthouse --help`

Reply:
done!